app/controllers/admin/promotional_features_controller.rb

```diff
@@ -60,8 +62,8 @@ def update_order
 private
 
   def get_layout
-    case action_name
-    when "reorder", "update_order"
+    design_system_actions = %w[edit reorder update_order]
```

Comment:
We need to add update to or when it rerenders the page we get this 
<img width="707" alt="image" src="https://user-images.githubusercontent.com/42515961/236143085-9bfaac02-65ae-4457-a103-64934d7d8bbd.png">

```
    design_system_actions = %w[edit update reorder update_order]

```